modules/cells/src/main/java/dmg/cells/nucleus/CellAdapter.java

```diff
@@ -75,7 +75,7 @@
 public class CellAdapter
     implements Cell, CellEventListener, CellEndpoint
 {
-    private static final Logger _log =
+    private static final Logger logger =
```

Comment:
If you want to fix naming issues, then please use UPPER case variable names for constants, like **static final**